tools/README.md

```diff
@@ -0,0 +1,12 @@
+# Enhancement Tools
```

Comment:
I think the commit message footer is spurious from a squash?

Reply:
Thanks, fixed!